api/resolvers/wallet.js

```diff
@@ -318,9 +318,10 @@ export default {
         where: {
           userId: me.id
         },
-        orderBy: {
-          createdAt: 'asc'
-        }
+        orderBy: [
+          { createdAt: 'desc' },
+          { id: 'desc' }
+        ]
```

Comment:
Logs have same timestamp within the same tx so I use `id` as a tie breaker for the real log insertion order with new logs first.